router/src/main/resources/Dockerfile

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 # -----------------------------------------------------------------------
 
-FROM envoyproxy/envoy:v1.16.0
+FROM envoyproxy/envoy-alpine:v1.16.2
```

Comment:
Can we upgrade to latest as in https://github.com/wso2/product-microgateway/pull/1786/files, so we can close the 1768 PR and merge this

Reply:
Upgrade with f42a48093b07ac76356bf50bbbe19cb0cfb3899c